package/native-package/package.json

```diff
@@ -16,14 +16,14 @@
   "peerDependencies": {
     "@bam.tech/react-native-image-resizer": ">=3.0.10",
     "@react-native-camera-roll/camera-roll": ">=5.0.0",
-    "@react-native-clipboard/clipboard": "^1.11.1",
+    "@react-native-clipboard/clipboard": ">=1.14.1",
     "react-native": ">=0.73.6",
     "react-native-audio-recorder-player": ">=3.6.4",
+    "react-native-blob-util": ">=0.19.9",
     "react-native-document-picker": ">=9.0.1",
-    "react-native-fs": ">=2.16.6",
     "react-native-haptic-feedback": ">=1.11.0",
     "react-native-image-crop-picker": ">=0.33.2",
-    "react-native-share": ">=4.1.0",
+    "react-native-share": ">=10.2.1",
```

Comment:
How do we ensure we are compatible with the future major versions of these packages? Shall we switch to caret (`^10.2.1`) ranges instead? The same applies to the other dependencies using the `>=` range.

Reply:
I agree with this. This has been an issue in the past as well, where integrators used a major release without us being notified about the changes and they faced issues. I don't have an idea why >= was taken up here in the past, but what you are suggesting makes sense.

Reply:
Well, let's fix this now. We are releasing breaking changes anyway :)

Reply:
done